### ISSUE TITLE
kStep termination condition is incorrect

### ISSUE THREAD

**Issue by bbalaji-ucsd** [mlab-upenn/pyEp#2]
I found that kStep < MAXSTEPS condition did not work well for me.
https://github.com/mlab-upenn/pyEp/blob/4b51e60c06b0e2e0b926a95e8bcf9f8838d79a08/pyEp/example.py#L27

I had to run two more steps for the program to terminate properly. Without that I get an error:

strtol caused error.: Invalid argument
strtol was called with strtol(
, 
, 10)
Error: The server closed the socket while the client was reading.

**Comment by dnong** [comment]
I have not encountered this error anywhere in my testing. 288 time steps should be correct, as that is the number of 5 minute intervals for a full day.

Could you give more details on the configuration you are running this?

**Comment by dnong** [comment]
You are correct, although I'm not entirely sure why that is the case with energyplus. The example has been updated to include this.
